packages/sn-filter-pane/src/components/ListboxContainer.tsx

```diff
@@ -42,12 +44,12 @@ const ListboxContainer = ({
       __DO_NOT_USE__: {
         selectDisabled: () => !allowSelect, // can we hook this into the selections api?
       },
-      ...options.listboxOptions,
+      ...options as IListBoxOptions,
```

Comment:
```suggestion
      direction: options.direction,
```
`options` are all the filterpane options, only pass specific options to the listbox

Reply:
this one still works but it triggers a warning from Typescript though, as options.direction is not in IUseOptions